cpp/ps0000/main (1).cpp

```diff
@@ -1,11 +1,12 @@
-#inline <iostream>
+#include <iostream> // 2/2
 using namespace std;
-int main()
+int main() // 1/1
 {
-    :cout << "Hallo world";
-//@ cannot be used instead of # 
-// cannot be used int main(void)
-//cannot be used iostream.h
-//Not 1 but 0 
-return 0
-}
\ No newline at end of file
+    cout << "Hello, World\n"; // 2/4
+    //@ cannot be used instead of # 
+    // cannot be used int main(void)
+    //cannot be used iostream.h
+    //Not 1 but 0 
+    return 0 // 1/1
+}
+// great work 😎 9/10 ⭐
```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit
+1 bonus mark for your comments